pkg/lib/cockpit-components-terminal.jsx

```diff
@@ -130,6 +138,37 @@ export class Terminal extends React.Component {
         window.removeEventListener('resize', this.onWindowResize);
     }
 
+    setText() {
+        try {
+            navigator.clipboard.readText()
+                    .then((text) => {
+                        this.props.channel.send(text);
+                    })
+                    .catch(() => {
+                        console.error('Text could not be pasted, use Shift+Insert');
```

Comment:
Let's write the actual error as well. Or is it always clear (between multiple browsers) why this fails?